packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 46.9.0 - August 8, 2019
+
+## Added
+
+#### salesforcedx-utils-vscode
+
+- Use VS Code api to open browser in host machine when operating in container mode ([PR #1448](https://github.com/forcedotcom/salesforcedx-vscode/pull/1448))
```

Comment:
@praksb, while reviewing a different message Nate mentioned that container refers to a very specific situation and he would prefer headless instead. Let me know if this looks ok:
Use VC Code API to open a browser in the host machine when operating in headless environment.

Reply:
that sounds good to me. i'll update the comment.